v-next/hardhat/src/internal/cli/init/prompt.ts

```diff
@@ -0,0 +1,90 @@
+import type { Template } from "./template.js";
+
+import { HardhatError } from "@ignored/hardhat-vnext-errors";
+
+export async function promptForWorkspace(): Promise<string> {
```

Comment:
I think this function would be better if it is aware of CWD and handles that instead, leading to a nicer user experience.

I guess this would mean two things:

1) We use `"."` as `initial`. ie. the relative path to CWD from CWD.
2) When we get a response, we do `resolveFromRoot(worspaceResponse, process.cwd())`. — This function is in hh-utils.

Reply:
Good suggestion, I updated the prompt to use paths relative to current working directory. Thanks :)